comments-bundle/src/Resources/contao/models/CommentsNotifyModel.php

```diff
@@ -15,16 +15,16 @@
 /**
  * Reads and writes comments subscriptions
  *
- * @property integer        $id
- * @property integer        $tstamp
- * @property string         $source
- * @property integer        $parent
- * @property string         $name
- * @property string         $email
- * @property string         $url
- * @property integer        $addedOn
- * @property string|boolean $active
- * @property string         $tokenRemove
+ * @property integer $id
+ * @property integer $tstamp
+ * @property string  $source
+ * @property integer $parent
+ * @property string  $name
+ * @property string  $email
+ * @property string  $url
+ * @property integer $addedOn
+ * @property boolean $active
```

Comment:
are these really booleans now? Not integers?

Reply:
Yes, see https://github.com/contao/contao/pull/4800